polynote-kernel/src/main/scala/polynote/messages/Messages.scala

```diff
@@ -274,6 +271,10 @@ abstract class NotebookUpdateCompanion[T <: NotebookUpdate](msgTypeId: Byte) ext
   implicit final val updateDiscriminator: Discriminator[NotebookUpdate, T, Byte] = Discriminator(msgTypeId)
 }
 
+
```

Comment:
I just moved this down because I was considering making it a subtype of `NotebookUpdate`. But we would need to distinguish between udpates that care about version and those that don't (e.g. since results only come from the kernel, they don't affect the version since that's an editing concern)